src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+export default configureStore({
```

Comment:
- [ ] Kindly create a store `const store = configureStore({` and `export default store` instead of configureStore 👍🏿